bin/workflows/pycbc_make_coinc_search_workflow

```diff
@@ -418,20 +418,31 @@ det_summ = [(s, r[0] if len(r) != 0 else None)]
 layout.two_column_layout(rdir['detector_sensitivity'],
                      det_summ + list(layout.grouper(r2, 2)))
 
+
 # run minifollowups on the output of the loudest events
+try:
```

Comment:
Why is this option in the top level workflow? The option name seems to indicate it should be in the set_foreground_minifollowups function.

Reply:
Similar for the sections below.  I think no changes to the top level workshop are really required here.

Reply:
@ahnitz because the change needs to be set in the top level workflow as it's a directive to the *planning* of the minifollowup workflow.

Reply:
So the change actually affects `main.dax`.

Reply:
Oh, wait, I see what you mean. Yes, let me fix that.

Reply:
@duncan-brown It looks to me like you are just passing this to each function. It has not effect on the top-level workflow script.

Reply:
Agreed, but it belongs in the subfunction `setup_foreground_minifollowups` not in the main dax. For e.g. I use that function in a second workflow, (basically a single-detector detchar model), if you put this in that function, my workflow generator (and main.dax) picks this up, but in the top-level code it does not.

Reply:
Basically check the config file at this point:

https://github.com/gwastro/pycbc/pull/2418/files#diff-6d5d67ddd622a3b42cb10c895f203f90R115

Reply:
@spxiwh yep, I'd misunderstood Alex's comment. That's what I'm doing.

Reply:
👍 (Looks like you'd already figured this out while I was typing :-) )

Reply:
And with the magic of force push, I did it the right way all along...